packages/babylon/src/tokenizer/index.js

```diff
@@ -683,89 +697,89 @@ export default class Tokenizer extends LocationParser {
         this.finishToken(tt.at);
         return;
 
-      case 96: // '`'
+      case charCodes.graveAccent:
         ++this.state.pos;
         this.finishToken(tt.backQuote);
         return;
 
-      case 48: {
+      case charCodes.digit0: {
         // '0'
         const next = this.input.charCodeAt(this.state.pos + 1);
         // '0x', '0X' - hex number
-        if (next === 120 || next === 88) {
+        if (next === charCodes.lowercaseX || next === charCodes.uppercaseX) {
           this.readRadixNumber(16);
           return;
         }
         // '0o', '0O' - octal number
-        if (next === 111 || next === 79) {
+        if (next === charCodes.lowercaseO || next === charCodes.uppercaseO) {
           this.readRadixNumber(8);
           return;
         }
         // '0b', '0B' - binary number
-        if (next === 98 || next === 66) {
+        if (next === charCodes.lowercaseB || next === charCodes.uppercaseB) {
           this.readRadixNumber(2);
           return;
         }
       }
       // Anything else beginning with a digit is an integer, octal
       // number, or float.
-      case 49:
-      case 50:
-      case 51:
-      case 52:
-      case 53:
-      case 54:
-      case 55:
-      case 56:
-      case 57: // 1-9
+      case charCodes.digit1:
+      case charCodes.digit2:
+      case charCodes.digit3:
+      case charCodes.digit4:
+      case charCodes.digit5:
+      case charCodes.digit6:
+      case charCodes.digit7:
+      case charCodes.digit8:
+      case charCodes.digit9:
         this.readNumber(false);
         return;
 
       // Quotes produce strings.
-      case 34:
-      case 39: // '"', "'"
+      case charCodes.questionMark:
+      case charCodes.apostrophe:
```

Comment:
That error is because here it should be `"` and `'`, not `?` and `'`.

Reply:
Good catch @nicolo-ribaudo, thanks you so much!